### PR TITLE
opengl: use new system helpers

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **opengl/system**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — APPROVED** [review]
LGTM

**Review by jwillikers — APPROVED** [review]

**Comment by prince-chrismc** [comment]
#12888 maybe 🤞

File: recipes/opengl/all/conanfile.py [hunk 1]
```diff
@@ -1,7 +1,10 @@
 from conan import ConanFile
 from conan.errors import ConanException
+from conan.tools.system import package_manager
 from conans import tools
```
Comment (prince-chrismc) [inline recipes/opengl/all/conanfile.py:4]: Is there a migration path for this?

Comment (ericLemanissier) [reply recipes/opengl/all/conanfile.py:4]: Yes, xorg/system did it already

Comment (prince-chrismc) [reply recipes/opengl/all/conanfile.py:4]: sweet taking a look!

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**